server/src/main/java/com/paypal/selion/grid/servlets/ProxyInfo.java

```diff
@@ -0,0 +1,249 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2015 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.grid.servlets;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.gson.JsonObject;
+
+import org.openqa.grid.common.RegistrationRequest;
+import org.openqa.grid.internal.RemoteProxy;
+import org.openqa.grid.internal.TestSlot;
+import org.openqa.selenium.remote.CapabilityType;
+
+import com.paypal.selion.node.servlets.LogServlet;
+import com.paypal.selion.proxy.SeLionRemoteProxy;
+
+@SuppressWarnings("unused")
+final class ProxyInfo {
+    /**
+     * Information not available
+     */
+    private static final String NOT_AVAILABLE = "not available";
+
+    /**
+     * Proxy is online
+     */
+    private static final String ONLINE = "online";
+
+    /**
+     * Proxy is offline
+     */
+    private static final String OFFLINE = "offline";
+
+    /**
+     * The url to view logs (via LogServlet) on the proxy. Defaults to {@value #NOT_AVAILABLE}
+     * 
+     * @see SeLionRemoteProxy#canViewLogs()
+     */
+    private String logsLocation;
+
+    /**
+     * @see RemoteProxy#isBusy()
+     */
+    private boolean isBusy;
+
+    /**
+     * @see RemoteProxy#getResourceUsageInPercent()
+     */
+    private float percentResourceUsage;
+
+    /**
+     * @see RemoteProxy#getTotalUsed()
+     */
+    private int totalUsed;
```

Comment:
We seem to be duplicating several grid node properties that are really part of RemoteProxy here.  I would leave RemoteProxy as the "record keeper" here on node status with regard to these properties.  We should offer simply reading and returning the values from RemoteProxy here but the "system of record"  on node status should be the RemoteProxy instance.